### PR TITLE
added header title under the logo

### PR DESCRIPTION
**Pull request by lebeza**
@shashank-naik @wansooseo 
- the added title under the logo.
image

### CODE REVIEW

File: web/themes/custom/digital_toolkit/templates/system/page.html.twig [hunk 1]
```diff
@@ -83,7 +83,15 @@
                                         </div>
                                     </div>
                                 </div>
-
+                                <div class="site-title-banner">
+                                    <div class="container">
+                                        <div class="row">
+                                            <div class="col">
+                                                <h1 class="site-title">Cancer Center Smoking Cessation Digital Toolkit </h1>
+                                            </div> <!-- /.col -->
```
Comment (wansooseo) [inline web/themes/custom/digital_toolkit/templates/system/page.html.twig]: Please remove the comment

File: web/themes/custom/digital_toolkit/templates/system/page.html.twig [hunk 2]
```diff
@@ -83,7 +83,15 @@
...
+                                        </div> <!-- /.row -->
```
Comment (wansooseo) [inline web/themes/custom/digital_toolkit/templates/system/page.html.twig]: Please remove the comment

File: web/themes/custom/digital_toolkit/templates/system/page.html.twig [hunk 3]
```diff
@@ -83,7 +83,15 @@
...
+                                    </div> <!-- /.container -->
```
Comment (wansooseo) [inline web/themes/custom/digital_toolkit/templates/system/page.html.twig]: Please remove the comment

File: web/themes/custom/digital_toolkit/src/styles/layout/_header.scss [hunk 4]
```diff
@@ -49,6 +49,21 @@ header {
   .logo-bar a.name.navbar-brand {
     display: none;
   }
+  .site-title-banner {
+    background-color: #f2f2f2;
```
Comment (wansooseo) [inline web/themes/custom/digital_toolkit/src/styles/layout/_header.scss]: Please use the variable. If it does not exist, please create one in `variables`

File: web/themes/custom/digital_toolkit/src/styles/layout/_header.scss [hunk 5]
```diff
@@ -49,6 +49,21 @@ header {
...
+    height: auto;
+  }
+  .site-title {
+    font-size: 15px;
+    font-weight: 700;
+    line-height: 1.2;
+    padding-top: 11px;
+    padding-left: 18px;
+    letter-spacing: 0.06em;
+    color: #272727;
```
Comment (wansooseo) [inline web/themes/custom/digital_toolkit/src/styles/layout/_header.scss]: Please use the variable. If it does not exist, please create one in `variables`

**Comment by lebeza** [comment]
@wansooseo please pull. All the comments are included in my latest push

**Pull request merged**